lib/backend/test/suite.go

```diff
@@ -322,52 +321,39 @@ func (s *BackendSuite) KeepAlive(c *check.C) {
 	c.Assert(err, check.IsNil)
 	defer watcher.Close()
 
-	item := backend.Item{Key: prefix("key"), Value: []byte("val1"), Expires: addSeconds(time.Now(), 2)}
-	lease, err := s.B.Put(ctx, item)
-	c.Assert(err, check.IsNil)
-
-	time.Sleep(time.Second)
-
-	// make sure that the value has not expired
-	out, err := s.B.Get(ctx, item.Key)
-	c.Assert(err, check.IsNil)
-	c.Assert(string(out.Value), check.Equals, string(item.Value))
-	c.Assert(string(out.Key), check.Equals, string(item.Key))
-
-	err = s.B.KeepAlive(ctx, *lease, addSeconds(time.Now(), 2))
-	c.Assert(err, check.IsNil)
+	expiresAt := addSeconds(s.Clock.Now(), 2)
+	item, lease := s.addItem(context.TODO(), c, prefix("key"), "val1", expiresAt)
 
-	// should have expired if not kept alive
-	diff := time.Until(addSeconds(time.Now(), 1))
-	time.Sleep(diff + 100*time.Millisecond)
+	s.Clock.Advance(1 * time.Second)
 
-	out, err = s.B.Get(ctx, item.Key)
+	// Move the expiration further in the future to avoid processing
+	// skew and ensure the item is available when we delete it.
+	// It does not affect the running time of the test
+	updatedAt := addSeconds(s.Clock.Now(), 60)
+	err = s.B.KeepAlive(context.TODO(), lease, updatedAt)
 	c.Assert(err, check.IsNil)
-	c.Assert(string(out.Value), check.Equals, string(item.Value))
-	c.Assert(string(out.Key), check.Equals, string(item.Key))
 
+	// Since the backend translates absolute expiration timestamp to a TTL
+	// and collecting events takes arbitrary time, the expiration timestamps
+	// on the collected events might have a slight skew
 	events := collectEvents(c, watcher, 3)
-	c.Assert(events[1].Type, check.Equals, backend.OpPut)
-	c.Assert(string(events[1].Item.Key), check.Equals, string(item.Key))
-	c.Assert(events[2].Type, check.Equals, backend.OpPut)
-	c.Assert(string(events[2].Item.Key), check.Equals, string(item.Key))
-
-	c.Assert(string(events[1].Item.Value), check.Equals, string(item.Value))
-	c.Assert(string(events[2].Item.Value), check.Equals, string(item.Value))
-
-	c.Assert(events[1].Item.Expires.IsZero(), check.Not(check.Equals), true, check.Commentf("expected non zero expiration time"))
-	c.Assert(events[2].Item.Expires.IsZero(), check.Not(check.Equals), true, check.Commentf("expected non zero expiration time"))
-	c.Assert(events[2].Item.Expires.After(events[1].Item.Expires), check.Equals, true, check.Commentf("expected %v after %v", events[2].Item.Expires, events[1].Item.Expires))
+	fmt.Println("Recv events:", events)
```

Comment:
```suggestion
	c.Log("Recv events: ", events)
```

Reply:
That's a debugging left-over - will remove.